sklearn/impute.py

```diff
@@ -99,27 +93,23 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
 
         - If X is not an array of floating values;
         - If X is sparse and `missing_values=0`;
-        - If `axis=0` and X is encoded as a CSR matrix;
-        - If `axis=1` and X is encoded as a CSC matrix.
+        - Otherwise X is encoded as a CSC matrix;
```

Comment:
I don't think it's right, simply removing the line with ``axis=1`` seems enough from my side.

Reply:
uhm but then there is no `axis` parameter anymore

Reply:
@glemaitre ``Otherwise`` seems strange from my side. We are saying : ``In the following cases : if ... if...``, so maybe ``if X is encoded as a CSR matrix``?

Reply:
oh I see yest you right